### DISCUSSION TITLE
How can I query all database schema?

### DISCUSSION THREAD

**Discussion by itzomen** [django-tenants/django-tenants#656] [Q&A]
Is there a way to query data from all the schemas?

My current approach is to modify the subdomain of the endpoint for each schema.
Wanted to know if there's a better way to do that.
Didn't find anything on stackoverflow

**Comment by tomturner** [comment]
You can only do this with raw sql and it a pain. Try goggling how to query across multi schemas

**Comment by itzomen** [comment]
Oh I see, sounds instereting..
Thanks will experiment on that
